### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nbouvrette**
Great plugin! I just thought that adding a Prettier Overrides section might help a few people who might not want to have line wrap. Otherwise having it as a plugin option might be an interesting feature if this is even feasible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,20 @@ Valid options:<br>
 | Default | CLI Override              | API Override            |
 | ------- | ------------------------- | ----------------------- |
 | `" = "` | `--key-separator "<sep>"` | `keySeparator: "<sep>"` |
+
+## Prettier Overrides
+
+By default, the plugin will use your Prettier configuration. For example that the lines will automatically wrap based on the `printWidth` option. To disable this you can simply add the following Prettier override:
```
Comment (eemeli) [inline README.md]:
```suggestion
### Using Overrides

To apply separate Prettier configuration for different file types,
you may use [configuration overrides](https://prettier.io/docs/en/configuration.html#configuration-overrides).
For example:
```

**Review by eemeli — CHANGES_REQUESTED** [review]
Sure, adding a mention about config overrides makes sense. Maybe though with slightly terser language, and a link to the upstream docs?

**Review by eemeli — APPROVED** [review]

**Comment by nbouvrette** [comment]
Looks good. By the way, I was thinking, it might be interesting at the top of the readme to add something like:

Once installed, you can fix your project's properties files by running

`npx eslint --ext .properties --fix .`

**Comment by nbouvrette** [comment]
Actually I forgot you could just run `npx prettier --write .` - maybe having an ESLint section might be helpful?

**Comment by eemeli** [comment]
I would think that the context here of this being a Prettier plugin can rely on Prettier's own docs providing usage guides. I would therefore prefer not to refer to other tools such as ESLint which are able to call Prettier.

**Comment by nbouvrette** [comment]
Ok so let's pretend I didn't completely mix ESLint and Prettier :) You don't want to fix properties file with ESLint! It worked for some reason in my confusion but running `npx prettier --write .` is the way to go. Sorry about this and thank for the plugin and merging the PR!

**Pull request merged**